app/Providers/AppServiceProvider.php

```diff
@@ -11,14 +12,22 @@ class AppServiceProvider extends ServiceProvider
      */
     public function register(): void
     {
-        //
     }
 
     /**
      * Bootstrap any application services.
      */
     public function boot(): void
     {
-        //
+        View::composer('*', function ($view) {
```

Comment:
There is a better way to solve this, without the need to edit a lower-level Laravel component for this. Consider using a Blade Component.